index.html

```diff
@@ -43,7 +43,7 @@
                 <li><a href="#news">News</a></li>
                 <li><a href="pages/Contact-Us/contact.html">Contact us</a></li>
                 <li><a href="pages/Blog/blog.html">Blogs</a></li>
-                <li><a href="#">DONATE</a></li>
+                <li><a href="../../Assets/Donate.html">DONATE</a></li>
```

Comment:
<code>./Assets/Donate.html</code>
Try to change href with this and recheck all other links as well.